syntaxes/wdl.tmGrammar.json

```diff
@@ -2,46 +2,92 @@
   "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
   "name": "WDL",
   "patterns": [
-    { "include": "#single-number-sign-comment" },
-    { "include": "#double-number-sign-comment" },
+    {
```

Comment:
FWIW, I ran `prettier` on the code base a couple of commits ago, but it's wanting to reformat things now. I'll add a CI check to ensure we don't keep flip-flopping.